gcn/notices/fermi/gbm/all.schema.json

```diff
@@ -0,0 +1,126 @@
+{
+  "$id": "https://gcn.nasa.gov/schema/main/gcn/notices/fermi/gbm/all.schema.json",
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "title": "GCN Schema for Fermi GBM Notices",
+  "description": "This schema is for the following Fermi GBM notices: alert, flight position, ground position, final position and subthreshold.",
+  "type": "object",
+  "allOf": [
+    {
+      "$ref": "../../core/AdditionalInfo.schema.json"
+    },
+    {
+      "$ref": "../../core/Alert.schema.json"
+    },
+    {
+      "$ref": "../../core/DateTime.schema.json"
+    },
+    {
+      "$ref": "../../core/Event.schema.json"
+    },
+    {
+      "$ref": "../../core/GeoLocBase.schema.json"
+    },
+    {
+      "$ref": "../../core/HardnessRatio.schema.json"
+    },
+    {
+      "$ref": "../../core/Localization.schema.json"
+    },
+    {
+      "$ref": "../../core/Reporter.schema.json"
+    },
+    {
+      "$ref": "../../core/Statistics.schema.json"
+    },
+    {
+      "$ref": "../../core/spectral/GammaRay.schema.json"
+    }
+  ],
+  "properties": {
+    "detectors_status": {
+      "type": "array",
+      "items": {
+        "$ref": "../../core/DetectorStatus.schema.json"
+      },
+      "minItems": 14,
+      "maxItems": 14,
+      "description": "Triggered detectors"
+    },
+    "duration_class": {
+      "enum": ["SHORT", "LONG", "UNCERTAIN"],
+      "description": "Long vs. short event"
+    },
+    "earth_angle": {
+      "type": "number",
+      "description": "Event-Earth angle [deg]"
+    },
+    "galactic_b": {
+      "type": "number",
+      "description": "Galactic altitude [deg]"
+    },
+    "galactic_l": {
+      "type": "number",
+      "description": "Galactic longitude [deg]"
+    },
+    "ground_generated_notice": {
+      "type": "boolean",
+      "description": "Some notices, designed to be generated in flight, can be issued on the ground."
```

Comment:
This is a confusing description. Do you just want an enum for flight vs ground generated?